### PR TITLE
fix: fix translations for mod updater

### PR DESCRIPTION
**Pull request by Scrumplex**
CC @flowln

### CODE REVIEW

File: launcher/ui/dialogs/ModUpdateDialog.cpp [hunk 1]
```diff
@@ -158,8 +158,8 @@ void ModUpdateDialog::checkCandidates()
             if (!reason.isEmpty())
                 text += tr("Reason: %1").arg(reason) + "<br>";
             if (!recover_url.isEmpty())
-                text += tr("Possible solution: ") + tr("Getting the latest version manually:") + "<br>" +
-                        QString("<a href='%1'>").arg(recover_url.toString()) + recover_url.toString() + "</a><br>";
+                text += tr("Possible solution: Getting the latest version manually:<br>"
+                           "<a href='%1'>%1</a><br>").arg(recover_url.toString());
```
Comment (flowln) [inline launcher/ui/dialogs/ModUpdateDialog.cpp]: ehm i don't really like putting the HTML tags inside the translation string, they are never translated so it's just more manual work on the translator's part :/

Comment (Scrumplex) [reply launcher/ui/dialogs/ModUpdateDialog.cpp]: I think we do that in most of the codebase, so it is probably fine?
I personally don't care much about this when translating to German

Comment (flowln) [reply launcher/ui/dialogs/ModUpdateDialog.cpp]: I think it's slightly annoying :tm:
no need to change it if you don't want to

**Review by flowln — APPROVED** [review]
engrish

**Review by timoreo22 — APPROVED** [review]
Get scrumped

**Review by txtsd — APPROVED** [review]

**Pull request merged**